deployment/network-operator/README.md

```diff
@@ -363,7 +363,7 @@ imagePullSecrets:
 | `ofedDriver.deploy` | bool | `false` | deploy Mellanox OFED driver container                                                                                                                                     |
 | `ofedDriver.repository` | string | `mellanox` | Mellanox OFED driver image repository                                                                                                                                     |
 | `ofedDriver.image` | string | `mofed` | Mellanox OFED driver image name                                                                                                                                           |
-| `ofedDriver.version` | string | `5.8-1.0.1.1.2` | Mellanox OFED driver version                                                                                                                                              |
+| `ofedDriver.version` | string | `5.9-0.5.6.0` | Mellanox OFED driver version                                                                                                                                              |
```

Comment:
can you also update version in project README ? just to keep things in sync

otherwise LGTM

Reply:
> can you also update version in project README ? just to keep things in sync

that's a really good catch! We didn't update README for a years